diff --git a/R/connection.R b/R/connection.R
index 8dc4573..e558635 100755
--- a/R/connection.R
+++ b/R/connection.R
@@ -781,6 +781,8 @@ h2o.resume <- function(recovery_dir=NULL) {
 # It will download a jar file if it needs to.
 #' @importFrom utils flush.console
 .h2o.downloadJar <- function(overwrite = FALSE) {
+  return(Sys.getenv("JARSOURCE"))
+
   old_options <- options(timeout = max(1000, getOption("timeout")))
   on.exit(options(old_options))
   if(!is.logical(overwrite) || length(overwrite) != 1L || is.na(overwrite)) stop("`overwrite` must be TRUE or FALSE")
