diff --git a/R/iriR.R b/R/iriR.R
index 1fdba29..675d2de 100644
--- a/R/iriR.R
+++ b/R/iriR.R
@@ -1,13 +1,22 @@
 # Loading data
 url <- paste0("https://warin.ca/datalake/iriR/iri_data.csv")
 path <- file.path(tempdir(), "temp.csv")
-if (httr::http_error(url)) { # network is down = message (not an error anymore)
+found = F
+if (tryCatch(httr::http_error(url), error=function(e) {F})) { # network is down = message (not an error anymore)
   message("No Internet connection or the server is in maintenance mode.")
   return(NULL)
 } else { # network is up = proceed to download via curl
   message("iriR: downloading remote dataset.")
-  with(options(timeout = max(300, getOption("timeout"))),curl::curl_download(url, path))
+  if (!tryCatch({
+	  with(options(timeout = max(300, getOption("timeout"))),curl::curl_download(url, path))
+	T}, error=function(e) {F})) {
+	   message("Failed to download remote dataset")
+	   return(NULL)
+  } else {
+   found = T
+}
 } # /if - network up or down
+if (found) {
 # Reading data
 csv_file <- file.path(paste0(tempdir(), "/temp.csv"))
 IRI_data <- read.csv(csv_file)
@@ -592,3 +601,11 @@ irir_visual <- function(country = "CAN", chart = "bar_1", title = TRUE, years =
     stop("invalid arguments")
   }
 }
+} else {  # fake it during package build / no internet
+ irir_company = F
+ irir_country = F
+ irir_data = F
+ irir_indicator = F
+ irir_industry = F
+ irir_visual = F
+}
