diff --git a/R/zzz.R b/R/zzz.R
index 12d6ed2..24a0280 100755
--- a/R/zzz.R
+++ b/R/zzz.R
@@ -5,11 +5,16 @@
 .onLoad <- function(libname, pkgname) {
     
     ns <- asNamespace( pkgname)
-    ah <- AnnotationHub::AnnotationHub()
-    metabolitesMapping <- ah[["AH91792"]]
-        
-    assign( "metabolitesMapping", metabolitesMapping, envir = ns)
+	tryCatch( {
+		ah <- AnnotationHub::AnnotationHub()
+		metabolitesMapping <- ah[["AH91792"]]
+		assign( "metabolitesMapping", metabolitesMapping, envir = ns)
+		namespaceExport( ns, "metabolitesMapping")
+	}, error = function (e) {
+		metabolitesMapping = F
+		assign( "metabolitesMapping", metabolitesMapping, envir = ns)
+        namespaceExport( ns, "metabolitesMapping")
+		warning("Could not connect annotation hub. metabolitesMapping is F")
+	})
     
-    namespaceExport( ns, "metabolitesMapping")
-    
-}
\ No newline at end of file
+}
