diff --git a/configure b/configure
index 51acf08..57f4bb8 100755
--- a/configure
+++ b/configure
@@ -69,9 +69,9 @@ elif [ -d "/usr/local/opt/nng" ]; then
 else
   echo "prior system libnng installation not found"
   echo "preparing to download and build library from source..."
-  curl -L https://api.github.com/repos/nanomsg/nng/tarball/$LIB_VER -o nng.tar.gz
-  gzip -dc nng.tar.gz | tar -xf -
-  rm -f nng.tar.gz
+  #curl -L https://api.github.com/repos/nanomsg/nng/tarball/$LIB_VER -o nng.tar.gz
+  gzip -dc $NNG_SRC | tar -xf -
+  #rm -f nng.tar.gz
   cd nanomsg-nng-$LIB_VER
   mkdir build
   cd build
@@ -118,9 +118,9 @@ if [ $? -ne 0 ]; then
   else
     echo "Attempt to use system libnng failed"
     echo "Preparing to download and build library from source..."
-    curl -L https://api.github.com/repos/nanomsg/nng/tarball/$LIB_VER -o nng.tar.gz
-    gzip -dc nng.tar.gz | tar -xf -
-    rm -f nng.tar.gz
+    #curl -L https://api.github.com/repos/nanomsg/nng/tarball/$LIB_VER -o nng.tar.gz
+    gzip -dc $NNG_SRC | tar -xf -
+    #rm -f nng.tar.gz
     cd nanomsg-nng-$LIB_VER
     mkdir build
     cd build
