diff --git a/R/zzz.R b/R/zzz.R
index 750fecd..f748615 100644
--- a/R/zzz.R
+++ b/R/zzz.R
@@ -1,6 +1,8 @@
 .onLoad <- function(libname, pkgname) {
+tryCatch({
   .games_nfl4th <- get_games_file()
   fd_model <- load_fd_model()
   assign(".games_nfl4th", .games_nfl4th, envir = parent.env(environment()))
   assign("fd_model", fd_model, envir = parent.env(environment()))
+}, error = function(e) { print(c("could not load nfl data", e))})
 }
