diff --git a/src/Makefile b/src/Makefile
index ac3358d..a0dd3c8 100755
--- a/src/Makefile
+++ b/src/Makefile
@@ -94,6 +94,7 @@ fastmap.o: bwa.h bntseq.h bwt.h bwamem.h kvec.h malloc_wrap.h utils.h kseq.h
 is.o: malloc_wrap.h
 kopen.o: malloc_wrap.h
 kstring.o: kstring.h malloc_wrap.h
+kthread.o: kthread.c malloc_wrap.h
 ksw.o: ksw.h neon_sse.h scalar_sse.h malloc_wrap.h
 main.o: kstring.h malloc_wrap.h utils.h
 malloc_wrap.o: malloc_wrap.h
