diff --git a/src/Makefile b/src/Makefile
index ac3358d..19eaf55 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -34,7 +34,7 @@ endif
 #.c.o:
 		#$(CC) -c $(CFLAGS) $(DFLAGS) $(INCLUDES) $< -o $@
 
-all:$(PROG) ../inst/bwa clean
+all:$(PROG) ../inst/bwa 
 
 ../inst/bwa: $(PROG)
 	mkdir -p ../inst
@@ -48,6 +48,7 @@ bwamem-lite:libbwa.a example.o
 		$(CC) $(CFLAGS) $(DFLAGS) example.o -o $@ -L. -lbwa $(LIBS)
 
 libbwa.a:$(LOBJS)
+		ls -la
 		$(AR) -csru $@ $(LOBJS)
 
 clean:
@@ -93,6 +94,7 @@ example.o: bwamem.h bwt.h bntseq.h bwa.h kseq.h malloc_wrap.h
 fastmap.o: bwa.h bntseq.h bwt.h bwamem.h kvec.h malloc_wrap.h utils.h kseq.h
 is.o: malloc_wrap.h
 kopen.o: malloc_wrap.h
+kthread.o: kthread.c malloc_wrap.h
 kstring.o: kstring.h malloc_wrap.h
 ksw.o: ksw.h neon_sse.h scalar_sse.h malloc_wrap.h
 main.o: kstring.h malloc_wrap.h utils.h
