diff --git a/src/scripts/vtk_download.sh b/src/scripts/vtk_download.sh
index dfd05ea..5f98391 100755
--- a/src/scripts/vtk_download.sh
+++ b/src/scripts/vtk_download.sh
@@ -3,14 +3,13 @@
 RSCRIPT_BIN=$1
 
 # Download VTK source
-${RSCRIPT_BIN} -e "utils::download.file(
-    url = 'https://www.vtk.org/files/release/9.0/VTK-9.0.1.tar.gz',
-    destfile = 'vtk-src.tar.gz')"
+# ${RSCRIPT_BIN} -e "utils::download.file(
+#     url = 'https://www.vtk.org/files/release/9.0/VTK-9.0.1.tar.gz',
+#     destfile = 'vtk-src.tar.gz')"
 
 # Uncompress VTK source
-${RSCRIPT_BIN} -e "utils::untar(tarfile = 'vtk-src.tar.gz')"
+${RSCRIPT_BIN} -e "utils::untar(tarfile = '${VTK_SRC}')"
 mv VTK-9.0.1 vtk-src
-rm -f vtk-src.tar.gz
 
 # Make ISO compilers happy
 echo 'typedef int make_iso_compilers_happy;' | cat - vtk-src/ThirdParty/expat/vtkexpat/lib/xmltok_impl.c > temp && mv temp vtk-src/ThirdParty/expat/vtkexpat/lib/xmltok_impl.c
