diff --git a/R/statcan_search.R b/R/statcan_search.R
index 5867164..b1cd532
--- a/R/statcan_search.R
+++ b/R/statcan_search.R
@@ -1,6 +1,7 @@
 # downloading the data file in English version
 url <- paste0("https://warin.ca/datalake/statcanR/statcan_data.qs")
 
+tryCatch({
 if (httr::http_error(url)) 
 { # network is down = message (not an error anymore)
   message("No tables with this combination of keywords")
@@ -10,7 +11,11 @@ if (httr::http_error(url))
   qs_file <- file.path(paste0(tempdir(), "/temp.qs"))
   statcandata <- qs::qread(qs_file)
   
+}},
+error=function(cond){
+	message("Could not download data")
 }
+)
 
 
 #' Searching function for statcanR
@@ -81,4 +86,4 @@ statcan_search <- function(keywords,lang) {
   }
 } 
 
-  
\ No newline at end of file
+  
