diff --git a/DESCRIPTION b/DESCRIPTION
old mode 100644
new mode 100755
diff --git a/LICENSE b/LICENSE
old mode 100644
new mode 100755
diff --git a/MD5 b/MD5
old mode 100644
new mode 100755
diff --git a/NAMESPACE b/NAMESPACE
old mode 100644
new mode 100755
diff --git a/NEWS.md b/NEWS.md
old mode 100644
new mode 100755
diff --git a/R/statCanR.R b/R/statCanR.R
old mode 100644
new mode 100755
diff --git a/R/statcan_search.R b/R/statcan_search.R
old mode 100644
new mode 100755
index 5867164..b1cd532
--- a/R/statcan_search.R
+++ b/R/statcan_search.R
@@ -1,6 +1,7 @@
 # downloading the data file in English version
 url <- paste0("https://warin.ca/datalake/statcanR/statcan_data.qs")
 
+tryCatch({
 if (httr::http_error(url)) 
 { # network is down = message (not an error anymore)
   message("No tables with this combination of keywords")
@@ -10,7 +11,11 @@ if (httr::http_error(url))
   qs_file <- file.path(paste0(tempdir(), "/temp.qs"))
   statcandata <- qs::qread(qs_file)
   
+}},
+error=function(cond){
+	message("Could not download data")
 }
+)
 
 
 #' Searching function for statcanR
@@ -81,4 +86,4 @@ statcan_search <- function(keywords,lang) {
   }
 } 
 
-  
\ No newline at end of file
+  
diff --git a/README.md b/README.md
old mode 100644
new mode 100755
diff --git a/build/vignette.rds b/build/vignette.rds
old mode 100644
new mode 100755
diff --git a/inst/README.Rmd b/inst/README.Rmd
old mode 100644
new mode 100755
diff --git a/inst/doc/statCanR.R b/inst/doc/statCanR.R
old mode 100644
new mode 100755
diff --git a/inst/doc/statCanR.Rmd b/inst/doc/statCanR.Rmd
old mode 100644
new mode 100755
diff --git a/inst/doc/statCanR.html b/inst/doc/statCanR.html
old mode 100644
new mode 100755
diff --git a/inst/doc/statcanR_example1.R b/inst/doc/statcanR_example1.R
old mode 100644
new mode 100755
diff --git a/inst/doc/statcanR_example1.Rmd b/inst/doc/statcanR_example1.Rmd
old mode 100644
new mode 100755
diff --git a/inst/doc/statcanR_example1.html b/inst/doc/statcanR_example1.html
old mode 100644
new mode 100755
diff --git a/inst/doc/statcanR_example2.R b/inst/doc/statcanR_example2.R
old mode 100644
new mode 100755
diff --git a/inst/doc/statcanR_example2.Rmd b/inst/doc/statcanR_example2.Rmd
old mode 100644
new mode 100755
diff --git a/inst/doc/statcanR_example2.html b/inst/doc/statcanR_example2.html
old mode 100644
new mode 100755
diff --git a/inst/doc/vignettes.R b/inst/doc/vignettes.R
old mode 100644
new mode 100755
diff --git a/inst/doc/vignettes.Rmd b/inst/doc/vignettes.Rmd
old mode 100644
new mode 100755
diff --git a/inst/doc/vignettes.html b/inst/doc/vignettes.html
old mode 100644
new mode 100755
diff --git a/man/figures/README-pressure-1.png b/man/figures/README-pressure-1.png
old mode 100644
new mode 100755
diff --git a/man/figures/shiny.png b/man/figures/shiny.png
old mode 100644
new mode 100755
diff --git a/man/statcan_data.Rd b/man/statcan_data.Rd
old mode 100644
new mode 100755
diff --git a/man/statcan_download_data.Rd b/man/statcan_download_data.Rd
old mode 100644
new mode 100755
diff --git a/man/statcan_search.Rd b/man/statcan_search.Rd
old mode 100644
new mode 100755
diff --git a/vignettes/statCanR.Rmd b/vignettes/statCanR.Rmd
old mode 100644
new mode 100755
diff --git a/vignettes/statcanR_example1.Rmd b/vignettes/statcanR_example1.Rmd
old mode 100644
new mode 100755
diff --git a/vignettes/statcanR_example2.Rmd b/vignettes/statcanR_example2.Rmd
old mode 100644
new mode 100755
diff --git a/vignettes/vignettes.Rmd b/vignettes/vignettes.Rmd
old mode 100644
new mode 100755
