diff --git a/configure b/configure
index bd423b3..4e114f0 100755
--- a/configure
+++ b/configure
@@ -85,7 +85,8 @@ cwd=`pwd`
 ## On Linux, download binary
 if [ ${platform} = "linux" ]; then
     cd inst/bin
-    download https://github.com/x13org/x13prebuilt/raw/master/v1.1.57/linux/${flavour}/x13ashtml
+    #download https://github.com/x13org/x13prebuilt/raw/master/v1.1.57/linux/${flavour}/x13ashtml
+    cp $X13BINARY x13ashtml
     chmod 0775 x13ashtml
     echo "*** downloaded Linux binary"
     cd ${cwd}
